doc/sphinx/source/releasenotes.rst

```diff
@@ -10,45 +10,70 @@ for each release of libCEED.
-`user manual <https://libceed.readthedocs.io/en/latest/>`_!
-And a C/Python interface, developed using the C Foreign Function Interface
-(`CFFI <https://cffi.readthedocs.io/en/latest/>`_)
-for Python. CFFI allows to reuse most of the C declarations and requires only a
-minimal adaptation of some of them. The C and Python APIs are mapped in a nearly
-1:1 correspondence. For instance, data stored in the CeedVector structure are
-associated to arrays defined via the numpy package. In fact, since libCEED heavily
-relies on pointers and arrays to handle the data, a Python structure that resembles
-the C arrays is needed. In details, numpy arrays allow this correspondence obtained
-by passing the numpy array memory address as pointers to the libCEED C API.
-
-Moreover, in this release, libCEED's suite of PETSc application examples has
-significantly expanded, including: An expanded Navier-Stokes miniapp, which now
-features an implicit time-integration formulation, SU/SUPG stabilization methods,
-free slip boundary conditions and quasi-2D computational domain for faster
-execution; A Continuum Mechanics example, which features different constitutive
-models, such as linear elasticity and hyperelasticity both at small and finite
-strain; Calculation of surface areas including the case of the surface of a
-cube and a cubed-sphere; And finally, the expansion of the set of PETSc Bakeoff
-Problems (BPs) on the cubed-sphere. In what follows, we provide a detailed
-description of the added examples.
-
-Backends available in this release, are the same ones available in :ref:`v0.5`.
-
+libCEED v0.6 contains numerous new features and examples, as well as expanded
+documentation in `this new website <https://libceed.readthedocs.io>`_.
+
+New features
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* New Python interface using `CFFI <https://cffi.readthedocs.io/>`_ provides a nearly
+  1-1 correspondence with the C interface, plus some convenience features.  For instance,
+  data stored in the :cpp:type:`CeedVector` structure are available without copy as
+  :py:class:`numpy.ndarray`.
+* Linear QFunctions can be assembled as block-diagonal matrices (per quadrature point,
+  :cpp:func:`CeedOperatorAssembleLinearQFunction`) or to evaluate the diagonal
+  (:cpp:func:`CeedOperatorAssembleLinearDiagonal`).  These operations are useful for
+  preconditioning ingredients and are used in the libCEED's multigrid examples.
+* The inverse of separable operators can be obtained using
+  :cpp:func`CeedOperatorCreateFDMElementInverse` and applied with
+  :cpp:func`CeedOperatorApply`.  This is a useful preconditioning ingredient, especially
+  for Laplacians and related operators.
+* New functions: :cpp:func:`CeedVectorNorm`, :cpp:func:`CeedOperatorApplyAdd`,
+  :cpp:func:`CeedQFunctionView`, :cpp:func:`CeedOperatorView`.
+* Make public accessors for various attributes to facilitate writing composable code.
+* New backend: ``/cpu/self/memcheck/serial``.
```

Comment:
Is this not the same `/cpu/self/ref/memcheck` that was already in v0.5?

Reply:
#418 is new in v0.6.